Week1/homework/8-type-checker.js

```diff
@@ -0,0 +1,128 @@
+// Declare 4 variables: 2 must be strings and 2 must be objects
+let a = {dessert: 'vla', price: '3$'};
+let b = {chocolate: 'bar', honey: 'jar'};
+let c = 'Amsterdam';
+let d = 'New York';
```

Comment:
You could have used a more meaningful variable name, let say `d` could have been `string_1` or `city_1` for example.

Reply:
Just a small note ;)